python/in_toto_attestation/v1/resource_descriptor.py

```diff
@@ -0,0 +1,30 @@
+#  Wrapper class for in-toto attestation ResourceDescriptor protos.
+
+import in_toto_attestation.v1.resource_descriptor_pb2 as rdpb
+from google.protobuf.struct_pb2 import Value
+import google.protobuf.json_format as pb_json
+import json
+
+class ResourceDescriptor:
```

Comment:
Is this strictly used for validation? So we shadow the pb.RD instance to validate its fields? Because the Statement class defined in this PR expects pb.RD in its constructor if I'm reading this right.

Reply:
Yes, this is strictly for validation. This sort of wrapping seems to be the regular practice for validating Python proto bindings, AFAIK.